SourceX/DiabloUI/diabloui.cpp

```diff
@@ -438,11 +370,142 @@ void LoadUiGFX() {
 	LoadArt("ui_art\\heros.pcx", &ArtHero, 4);
 }
 
+// clang-format off
+#define BLANKCOLOR { 0, 0xFF, 0, 0 }
+// clang-format on
+
+void LoadFallbackPalette()
+{
+	// clang-format off
+	PALETTEENTRY fallback_palette[256] = {

```

Comment:
Is this the Diablo palette? Are we allowed to use it?

Reply:
The original (I only converted the relevant parts) palettes is embedded in the application binary, so I would say this falls in the same category as the monster data table etc. It can be found in the .rc file from upstream so was already a part of the repo. Note that this pallet (even in full form) also only applies for the alert box, other palettes are used for the other screens and in-game.

Reply:
OK, makes sense